rpm/BUILD.bazel

```diff
@@ -1,133 +1,8 @@
+# @unused
 load("@bazeldnf//:def.bzl", "bazeldnf")
-load("@bazeldnf//:deps.bzl", "rpmtree", "tar2files")
-load("@rules_cc//cc:defs.bzl", "cc_library")
-
-cc_library(
-    name = "cryptsetup",
-    srcs = ["//rpm:cryptsetup-libs/usr/lib64"],
-    hdrs = ["//rpm:cryptsetup-libs/usr/include"],
-    linkstatic = 1,
-    strip_include_prefix = "/rpm/cryptsetup-libs/",
-    visibility = ["//visibility:public"],
-)
```

Comment:
Is this change related to go upgrade?

Reply:
This is a temporary change that I had to make since the RPMs are no longer available at the URLs that are hard coded here. The [bazel PR](https://github.com/edgelesssys/constellation/pull/1186) updates those URLs to be correct again.

Reply:
The cc_library depends on those RPMs so I just removed it here.